Bottom: d9a90893a4fc26e64719eb57f3e7d2e6f14b7b64
Top:    cd0a826a8ce4b542b8bbd552938cacb321436432
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-19 20:30:16 +0800

Refresh of vhost_net-introduce-helper-to

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 2127e8e..cac28fd 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -547,8 +547,7 @@ static void handle_tx(struct vhost_net *net)
 		/* Sanity check */
 		len = init_iov_iter(vq, &msg.msg_iter, hdr_size, out);
 		if (!len) {
-			vq_err(vq, "Unexpected header len for TX: "
-			"%zd expected %zd\n",
+			vq_err(vq, "Unexpected header len for TX: %zd expected %zd\n",
 			len, hdr_size);
 			break;
 		}
